documentation/README.md

```diff
@@ -1631,6 +1632,30 @@ map(shave(1, parseInt), [0, 1.1, 2.2]); // => [0, 1, 2]
 </sup></div>
 
 
+### signum
+
+Returns [`1` if `n` is positive, `-1` if `n` is negative and `0` if `n` is `0`](https://en.wikipedia.org/wiki/Signum_function). Otherwise returns `NaN`.
```

Comment:
I don't like the big link. How about: 

> Returns the [sign of a number](https://en.wikipedia.org/wiki/Signum_function). `1` if `n` is positive, `-1` if `n` is negative and `0` if `n` is `0`. Otherwise returns `NaN`.